rclpy/test/test_node.py

```diff
@@ -1896,6 +1896,18 @@ def test_static_dynamic_typing(self):
         self.assertEqual(self.node.get_parameter('int_param').value, 4)
         self.assertFalse(self.node.has_parameter('dynamic_param'))
 
+    def test_wait_for_node(self):
+        try:
+            node = rclpy.create_node(
+                'waiting_for_this_node', namespace='/my_ns', context=self.context
+            )
+            self.assertTrue(self.node.wait_for_node('/my_ns/waiting_for_this_node', 0.5))
```

Comment:
I would use a larger timeout, so it works reliably in all platforms and with all rmw implementations:

```suggestion
            self.assertTrue(self.node.wait_for_node('/my_ns/waiting_for_this_node', 3))
```

Reply:
done [90e8bf0](https://github.com/ros2/rclpy/pull/930/commits/90e8bf0f56903c40ee4a6741e85acbb5d5cedb0f)